### ISSUE TITLE
Add "Add to project" bulk operation on stream list

### ISSUE THREAD

**Issue by MrAlex8** [speckleworks/SpeckleAdmin#24]
In the speckle streams viewer, it would be nice to have a button/dropdown to 'add stream to project'

![image](https://user-images.githubusercontent.com/45394203/51192105-bb41c700-18dd-11e9-8d14-46dfe5c18819.png)

**Comment by didimitrie** [comment]
Yep, agreed - it would be useful. Would a drop down of existing projects suffice? Will see about it when we get to it...

**Comment by MrAlex8** [comment]
I think it would suffice

**Comment by didimitrie** [comment]
reopening as you've closed it by mistake?

**Comment by MrAlex8** [comment]
Precisely, mersi
